### ISSUE TITLE
No VirtualBox with Docker Toolkit

### ISSUE THREAD

**Issue by abdennour** [johanhaleby/docker-machine-port-forwarder#4]
Previously, Docker machine is running as VirtualBox in MacOS. Currently, nope nope.

Any alternative? Or can we make two versions of this `pf` script?

**Comment by johanhaleby** [comment]
I don't think you need explicit port forwarding like this anymore, at least not in my experience. Now you simply expose the port when starting the container and then you should be able to connect to the port.
